vm/ubpf_vm.c

```diff
@@ -302,7 +321,7 @@ ubpf_exec(const struct ubpf_vm* vm, void* mem, size_t mem_len, uint64_t* bpf_ret
 
     reg[1] = (uintptr_t)mem;
     reg[2] = (uint64_t)mem_len;
-    reg[10] = (uintptr_t)stack + sizeof(stack);
+    reg[10] = (uintptr_t)stack + UBPF_STACK_SIZE;
```

Comment:
If someone uses `207` as a `UBPF_STACK_SIZE` we would allocate

`(207 + 7) / 8` = 26 `uint64_t`-sized things

which would mean that `reg[10]` would not contain *exactly* the right cutoff. 

At least that's how it looks to me.

Reply:
Thank you!